.changeset/seven-monkeys-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/shopify-app-remix': patch
+---
+
+[AppProvider] - automatically handle the "shopify:navigate" event for Remix apps using Polaris Web Components
```

Comment:
Can we make it clear that if you are using Polaris React still, this won't have any effect and is none breaking?